src/toast/Toast.stories.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {storiesOf} from "@storybook/react";
+import {action} from "@storybook/addon-actions";
+import {TestToaster} from "./TestToaster";
+
+storiesOf("Toast", module).add("Toast Simple", () => (
+    <div>
+        <h4> Toast </h4>
+        <TestToaster />
```

Comment:
Here you are suppose to add code which shows how user will use the Toast component in other application.

Reply:
Addressed this comment.